code/game/turfs/simulated/walls_reinforced.dm

```diff
@@ -132,11 +132,11 @@
 			to_chat(user, "<span class='notice'>You pry off the cover.</span>")
 		if(RWALL_SHEATH)
 			to_chat(user, "<span class='notice'>You struggle to pry off the outer sheath...</span>")
-			if(!I.use_tool(src, user, 100, volume = I.tool_volume) || d_state != RWALL_SHEATH)
+			if(!I.use_tool(src, user, 100, volume = I.tool_volume))
 				return
-			to_chat(user, "<span class='notice'>You pry off the outer sheath.</span>")
-			dismantle_wall()
-			return
+			if(dismantle_wall())
```

Comment:
Note: this will mean that deconstructing a wall will not display a message until farie's fix PR is merged. 😄